### PR TITLE
feat: change cluster configuration

### PR DESCRIPTION
**Pull request by davidebianchi**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensible content has been committed

### CODE REVIEW

File: docs/development_suite/clusters-management/vendors-runtime-services.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 ---
 id: vendors-runtime-services
```
Comment (RiccardoCorona-MiaPlatform) [inline docs/development_suite/clusters-management/vendors-runtime-services.md:2]: L'ID e il nome del file lo lasciamo invariato?

File: docs/development_suite/clusters-management/vendors-runtime-services.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
-title: Vendors & Runtime services
-sidebar_label: Vendors & Runtime services
+title: Clusters Overview
+sidebar_label: Clusters
 ---
 
-When the user [connects a new cluster](/getting-started/handbooks/company/connect-and-manage-cluster.md#connect-a-cluster), they are required to enter information about the cluster's vendor and runtime service.
+Mia-Platform Console allows authorized users to manage the Kubernetes clusters on which projects are deployed. 
+
+## Supported clusters
```
Comment (RiccardoCorona-MiaPlatform) [inline docs/development_suite/clusters-management/vendors-runtime-services.md:9]: Non sarebbe meglio `Supported vendors and runtime services for a cluster` eventualmente?

File: docs/development_suite/clusters-management/vendors-runtime-services.md (hunk 1 above)
Comment (davidebianchi) [reply docs/development_suite/clusters-management/vendors-runtime-services.md:2]: Changed in overview

File: docs/development_suite/clusters-management/vendors-runtime-services.md (hunk 2 above)
Comment (davidebianchi) [reply docs/development_suite/clusters-management/vendors-runtime-services.md:9]: Changed

**Review by RiccardoCorona-MiaPlatform — APPROVED** [review]

**Pull request merged**